docs/self-managed/platform-deployment/helm-kubernetes/guides/multi-namespace-deployment.md

```diff
@@ -1,35 +1,124 @@
 ---
 id: multi-namespace-deployment
 title: "Multi-namespace deployment"
-description: "Deploy Camunda 8 Self-Managed across namespaces for better resources utilization and to reduce redundancy"
+description: "Deploy Camunda 8 Self-Managed across namespaces for better resource utilization and to reduce redundancy."
 ---
 
-You can deploy Camunda 8 Self-Managed across namespaces, where we have a single management deployment (Identity and Web Modeler), and multiple automation deployments (Zeebe, Operate, Tasklist, Optimize).
+Camunda 8 Self-Managed can be deployed across multiple namespaces. This configuration includes a single management deployment (comprising Console, Identity, and Web Modeler) and multiple automation deployments (including Zeebe, Operate, Tasklist, and Optimize).
 
-In this set up each namespace has its own Helm deployment independently, but each one uses a different values file based on if the deployment mode is `management` or `automation`.
 
-The following are three values files, one for the `managment` deployment and two for the `automation` deployment:
+In this setup, each namespace is deployed independently using Helm. The deployments are differentiated by their deployment mode - `management` or `automation` - and each utilizes a distinct values file tailored to its mode.
+
+In the following example, we use three Helm value files, one for the `management` deployment and two for the `automation` deployments:
+
+## Preparation
```

Comment:
I don't really follow this section. I feel it is missing an explanation of a scenario and a starting point.